app/views/content/what-pension-does-a-teacher-get.md

```diff
@@ -8,7 +8,7 @@ image: false
 promo_content:
     - content/train-to-be-a-teacher/promos/mailing-list-promo-salaries
 related_content:
-    Salaries and benefits : "/salaries-and-benefits"
+    Teacher pay and benefits : "/is-teaching-right-for-me/teacher-pay-and-benefits"
```

Comment:
Do we not want to move the pensions page under ITRFM as well? Or is there a further conversation to be had about this?

@philspray @EllieNodder